mongo-connector/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Note that this image requires a version of Mongodb of 3.6 or later
+FROM confluentinc/cp-kafka-connect:6.1.9
+
+COPY connect_start.sh /scripts/connect_start.sh
```

Comment:
Why do we choose the name sample_connect_start.sh rather than connect_start.sh? Also if sample_connect_start.sh is the preferred name outside container. Maybe update the copy command: 
```
COPY sample_connect_start.sh /scripts/connect_start.sh
```

Reply:
The original idea is that the sample_connect_start.sh would be a starting point to create a connect_start.sh. This seems slightly confusing after further thought, and instead I am going to push some changes that allow for attaching different connect_start.sh files via environmental variables rather than embedding it in the image.